src/Workspaces/CSharp/Portable/Rename/CSharpRenameRewriterLanguageService.cs

```diff
@@ -192,98 +166,6 @@ public RenameRewriter(RenameRewriterParameters parameters)
-            {
-                return !isInConflictLambdaBody &&
-                       _skipRenameForComplexification == 0 &&
-                       !_isProcessingComplexifiedSpans &&
-                       _conflictLocations.Contains(node.Span) &&
-                       (node is AttributeSyntax ||
-                        node is AttributeArgumentSyntax ||
-                        node is ConstructorInitializerSyntax ||
-                        node is ExpressionSyntax ||
-                        node is FieldDeclarationSyntax ||
-                        node is StatementSyntax ||
-                        node is CrefSyntax ||
-                        node is XmlNameAttributeSyntax ||
-                        node is TypeConstraintSyntax ||
-                        node is BaseTypeSyntax);
-            }
-
-            public override SyntaxToken VisitToken(SyntaxToken token)
-            {
-                var shouldCheckTrivia = _stringAndCommentTextSpans.ContainsKey(token.Span);
-                _isProcessingTrivia += shouldCheckTrivia ? 1 : 0;
-                var newToken = base.VisitToken(token);
-                _isProcessingTrivia -= shouldCheckTrivia ? 1 : 0;
-
-                // Handle Alias annotations
-                newToken = UpdateAliasAnnotation(newToken);
-
-                // Rename matches in strings and comments
-                newToken = RenameWithinToken(token, newToken);
-
-                // We don't want to annotate XmlName with RenameActionAnnotation
-                if (newToken.Parent.IsKind(SyntaxKind.XmlName))
-                {
-                    return newToken;
-                }
-
-                var isRenameLocation = IsRenameLocation(token);
-
-                // if this is a reference location, or the identifier token's name could possibly
-                // be a conflict, we need to process this token
-                var isOldText = token.ValueText == _originalText;
-                var tokenNeedsConflictCheck =
-                    isRenameLocation ||
-                    token.ValueText == _replacementText ||
-                    isOldText ||
-                    _possibleNameConflicts.Contains(token.ValueText) ||
-                    IsPossiblyDestructorConflict(token) ||
-                    IsPropertyAccessorNameConflict(token);
-
-                if (tokenNeedsConflictCheck)
-                {
-                    newToken = RenameAndAnnotateAsync(token, newToken, isRenameLocation, isOldText).WaitAndGetResult_CanCallOnBackground(_cancellationToken);
-
-                    if (!_isProcessingComplexifiedSpans)
-                    {
-                        _invocationExpressionsNeedingConflictChecks.AddRange(token.GetAncestors<InvocationExpressionSyntax>());
-                    }
-                }
-
-                return newToken;
-            }
-
-            private bool IsPropertyAccessorNameConflict(SyntaxToken token)
```

Comment:
This part of code are changed into static version and move to the bottom of the class

Reply:
Any chance we could make a commit here that moves these back? It's definitely making this diff much harder to follow.

Reply:
OK, I have pushed a commit to reorder this part.